components/validator/issuer.go

```diff
@@ -109,7 +109,7 @@ func reviveChain(issuingTime time.Time) (*iotago.Commitment, iotago.BlockID, err
 	apiForSlot := deps.Protocol.APIForSlot(lastCommittedSlot)
 
 	// Get a rootblock as recent as possible for the parent.
-	parentBlockID := iotago.EmptyBlockID
+	parentBlockID := iotago.EmptyBlockID //TODO: use GenesisBlockID
```

Comment:
Can it be resolved?

Reply:
I think so, I've created an accompanying PR in inx-validator. Conflicts on this file should be resolved by taking version from develop.